ui/packages/catalog-ui-search/src/main/webapp/component/list-editor/list-editor.view.js

```diff
@@ -12,6 +12,7 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/
+/* eslint-disable no-var */
```

Comment:
Why are we keeping vars in this file?

Reply:
several files couldn't be fixed automatically by eslint and I don't think fixing them is in scope for this change

Reply:
the test steps for this pr would need to change from `build` to `test every affected ui component thoroughly`

Reply:
It might be worth fixing these. This file is particularly weird because we have two vars with the same name :thinking:

Reply:
I agree, but I'd like to fix them in a series of isolated prs in case the fix causes a regression

Reply:
:+1: